app/scripts/lib/buy-url.js

```diff
@@ -23,12 +22,15 @@ const fetchWithTimeout = getFetchWithTimeout(SECOND * 30);
 /**
  * Create a Wyre purchase URL.
  *
- * @param {string} address - Ethereum destination address
+ * @param {string} walletAddress - Ethereum destination address
+ * @param {string} chainId - Current chain ID
  * @returns String
  */
-const createWyrePurchaseUrl = async (address) => {
-  const fiatOnRampUrlApi = `${SWAPS_API_V2_BASE_URL}/networks/${MAINNET_NETWORK_ID}/fiatOnRampUrl?serviceName=wyre&destinationAddress=${address}`;
-  const wyrePurchaseUrlFallback = `https://pay.sendwyre.com/purchase?dest=ethereum:${address}&destCurrency=ETH&accountId=AC-7AG3W4XH4N2&paymentMethod=debit-card`;
+const createWyrePurchaseUrl = async (walletAddress, chainId) => {
+  const { wyre: { srn, currencyCode } = {} } = BUYABLE_CHAINS_MAP[chainId];
```

Comment:
we could do the destructuring in two lines for better code readability. 
```suggestion
  const { wyre = {}} = BUYABLE_CHAINS_MAP[chainId];
  const { srn, currencyCode } = wyre;
 
```